Sources/CryptoSwift/CipherType.swift

```diff
@@ -12,20 +12,20 @@
     import Darwin
 #endif
 
-public enum CipherError: ErrorType {
+public enum CipherError: ErrorProtocol {
     case Encrypt
     case Decrypt
 }
 
-public protocol CipherType {
-    func cipherEncrypt(bytes: [UInt8]) throws -> [UInt8]
-    func cipherDecrypt(bytes: [UInt8]) throws -> [UInt8]
+public protocol Cipher {
```

Comment:
To follow the Swift 3.0 API guidelines

https://github.com/apple/swift-evolution/blob/master/proposals/0006-apply-api-guidelines-to-the-standard-library.md